source/channels/make-calls.rst

```diff
@@ -0,0 +1,86 @@
+Make a call (beta)
+==================
+
+|enterprise| |cloud| |self-hosted|
```

Comment:
This seems to say `enterprise`, but later down under 'configuration-settings.rst' it says `all-plans`.